packages/block-directory/src/store/test/reducer.js

```diff
@@ -17,46 +17,76 @@ import { installedItem, downloadableBlock } from './fixtures';
 describe( 'state', () => {
 	describe( 'downloadableBlocks()', () => {
 		it( 'should update state to reflect active search', () => {
-			const initialState = deepFreeze( {
```

Comment:
`deepFreeze`ing the initial state is helpful for checking against mutations, why remove it?

Reply:
I think it's still useful because it makes sure you're not accidentally mutating state in `downloadableBlocks`, so I'd prefer it stay in.

Reply:
Done.